libshaderc/CMakeLists.txt

```diff
@@ -14,10 +14,26 @@ target_link_libraries(shaderc PRIVATE
 target_link_libraries(shaderc PRIVATE shaderc_util)
 target_link_libraries(shaderc PRIVATE SPIRV)
 
+
 add_shaderc_tests(
   TEST_PREFIX shaderc
   LINK_LIBS shaderc
   INCLUDE_DIRS include ${glslang_SOURCE_DIR}
-  TEST_NAMES 
+  TEST_NAMES
     shaderc
     shaderc_cpp)
+
+
+if (NOT "${MSVC}")
+  # If we are not on windows, then create a single static library
```

Comment:
I wonder if we should make a README.md for libshaderc.  There are now two ways to use the library.